Sources/ProjectSpec/BuildRule.swift

```diff
@@ -87,6 +98,7 @@ extension BuildRule: JSONEncodable {
             "outputFiles": outputFiles,
             "outputFilesCompilerFlags": outputFilesCompilerFlags,
             "name": name,
+            "runOncePerArchitecture": runOncePerArchitecture,
```

Comment:
Could you add this to dict only if the value differs from `runOncePerArchitectureDefault`